code/modules/surgery/surgery_step.dm

```diff
@@ -132,7 +132,10 @@
 /datum/surgery_step/proc/play_preop_sound(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	if(!preop_sound)
 		return
-	playsound(get_turf(target), preop_sound, 75, TRUE, falloff_exponent = 12, falloff_distance = 1)
+	if(islist(preop_sound))
+		playsound(get_turf(target), preop_sound[tool.type], 75, TRUE, falloff_exponent = 12, falloff_distance = 1)
```

Comment:
Because you're checking for exact types, you won't get subtypes of the items you're looking for in the `preop_sound` list. 

What sound does it make when you use a laser scalpel or alien saw on this step? Because I have a feeling it may be "no sound", or may possibly runtime error

Reply:
Hmm, what would you recommend I do? reduce the obj path to give cover broader types?

Reply:
I'm not entirely certain, but you'd be looking at `istype()` checks to look for that tool's type and all subtypes

Reply:
maybe just handle this using an element, send a signal before preop on the surgery tool

Reply:
ok so i made it into a loop to check for subtypes and it returns a correct value, it just doesn't play the sound when it comes to playsound() for some reason